web/src/buddy_list.js

```diff
@@ -6,11 +6,14 @@ import render_presence_rows from "../templates/presence_rows.hbs";
 import * as blueslip from "./blueslip";
 import * as buddy_data from "./buddy_data";
 import * as message_viewport from "./message_viewport";
+import * as narrow_state from "./narrow_state";
 import * as padded_widget from "./padded_widget";
+import * as people from "./people";
 import * as scroll_util from "./scroll_util";
 
 class BuddyListConf {
-    container_sel = "#buddy-list-users-matching-view";
+    narrow_container_sel = "#buddy-list-users-matching-view";
```

Comment:
@timabbott would you prefer this be named as `matching_view_container_sel` or something?

Reply:
`matching_view_container_selector` seems good. `sel` is an abbreviation and thus violates our coding standards.